operations-api/build.gradle.kts

```diff
@@ -0,0 +1,49 @@
+import org.jetbrains.kotlin.gradle.plugin.mpp.apple.XCFramework
+
+plugins {
+    kotlin("multiplatform") version Versions.kotlin
+    id("io.kotest.multiplatform") version Versions.kotest apply false
```

Comment:
Removing this plugin causes:
Cannot add task 'commonizeNativeDistribution' as a task with that name already exists.
So I disabled it with false parameter

Reply:
Any ideas how to solve it in other way?

Reply:
Why would an API module know anything about any operators? In the linked diagram this module is independent from any module. Because of its independence there will probably be no concrete implementation/nothing to really test.